README.md

```diff
@@ -398,6 +398,40 @@ HTTP call to the platform:<br/>
   }
   ```
 
+### fetchDeviceDetails:
```

Comment:
Is there somewhere else it would make sense for this doc to live? Just want to be careful with what we are including in the top level read me so we don't overload it with information.

Reply:
yea. we added it under the request-override heading, we can move the complete requestoverride file somewhere else and provide a link here

Reply:
moved the requestOverrides to a different doc file inside Doc/  folder